docs/platform-support.rst

```diff
@@ -215,3 +215,33 @@ NVIDIA Network Operator has been validated in the following scenarios:
      - Yes
      - 
 
+
+=============================================
+Supported Precompiled Images for DOCA Drivers
+=============================================
+
+--------
+Overview
+--------
+
+To save startup (compilation) time and operational effort, precompiled DOCA driver images are available for common OS/flavor/kernel/architecture variants.
+
+The container image tag pattern used for common variants is: **driver_ver-container_ver-kernel_ver-flavor-os-arch**. For example: ``24.07-0.6.1.0-6.8.0-49-generic-ubuntu24.04-amd64``
+
+**NOTE:** The images are also packaged as multi-architecture manifests, so manually stating architecture component is optional.
```

Comment:
Not needed.
The only thing the user can do is specify the DOCA driver version in the NicClusterPolicy.
It cannot define kernel/arch or anything like that